Src/BootCamp.Chapter/Students/Student.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using BootCamp.Chapter.Subjects;
+using BootCamp.Chapter.Teachers;
+
+namespace BootCamp.Chapter.Students
+{
+    public abstract class Student : IStudent
```

Comment:
Not abstract, because a student can exist by itself. It doesn't have anything that is not unimplemented.